checkBitSize/checkBitSize.h

```diff
@@ -0,0 +1,60 @@
+/* Sebastian Burkhart, 2019 */
```

Comment:
If some of these declarations are from the original DataCompressor repository, would it not make more sense to incorporate some there and refer to this file (or to link it)? Otherwise, code is duplicated across two repositories and may be changed independently, making it harder to make it behave consistently

Reply:
Included the header provided by the DataCompressor, removed the duplicates.